src/error.rs

```diff
@@ -1,5 +1,7 @@
 // SPDX-License-Identifier: CC0-1.0
```

Comment:
The commit message still refers to iterators.

Reply:
Thanks, fixed.